importers/scripts/importSwordTest.ts

```diff
@@ -0,0 +1,39 @@
+import { join } from 'path';
+
+import { ModuleIndex, SwordModule } from '../src';
+import { readFileSync } from 'fs';
+
+const run = async () => {
+    // const creator = new BeDatabaseCreator({
```

Comment:
remove commented out code

Reply:
since this is a file for manual dev testing i think it can be justifiable to leave some commented code for easy switching

Reply:
would prefer to put in a conditional: "if (TESTING_BIBLE_ENGINE === true), run this code, etc

but up to you!

Reply:
done